src/ripple/overlay/impl/PeerImp.h

```diff
@@ -656,24 +649,13 @@ void
 PeerImp::sendEndpoints(FwdIt first, FwdIt last)
 {
     protocol::TMEndpoints tm;
-    for (; first != last; ++first)
+
+    while (first != last)
```

Comment:
Is there any advantage to using `while` vs `for` in this case?

Reply:
Not really.